package.json

```diff
@@ -30,9 +30,11 @@
     "@nestjs/platform-express": "^8.0.0",
     "@nestjs/typeorm": "^8.0.4",
     "argon2": "^0.28.5",
+    "cheerio": "^1.0.0-rc.12",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.13.2",
     "express": "^4.18.1",
+    "node-fetch": "2.0",
```

Comment:
How about using native fetch instead node-fetch?

Reply:
I use node 16 LTS. native fetch need version 18. Isn't it?

Reply:
You can use native fetch with `--experimental-fetch` option.
If you think it's dangerous to use experimental features, you might keep `node-fetch` as a dependency.

Reply:
I think it's good using `node-fetch`.